WordPressKit/SharingServiceRemote.swift

```diff
@@ -57,22 +57,22 @@ open class SharingServiceRemote: ServiceRemoteWordPressComREST {
                 }
 
                 let responseString = responseObject.description as NSString
-                let services = responseDict.forKey(ServiceDictionaryKeys.services) as NSDictionary
+                let services: NSDictionary = (responseDict.forKey(ServiceDictionaryKeys.services) as? NSDictionary) ?? NSDictionary()
 
                 let publicizeServices: [RemotePublicizeService] = services.allKeys.map { (key) -> RemotePublicizeService in
-                    let dict = services.forKey(key) as NSDictionary
+                    let dict = (services.forKey(key) as? NSDictionary) ?? NSDictionary()
                     let pub = RemotePublicizeService()
 
-                    pub.connectURL = dict.string(forKey: ServiceDictionaryKeys.connectURL)
-                    pub.detail = dict.string(forKey: ServiceDictionaryKeys.description)
-                    pub.externalUsersOnly = dict.number(forKey: ServiceDictionaryKeys.externalUsersOnly).boolValue
-                    pub.icon = dict.string(forKey: ServiceDictionaryKeys.icon)
-                    pub.serviceID = dict.string(forKey: ServiceDictionaryKeys.ID)
-                    pub.jetpackModuleRequired = dict.string(forKey: ServiceDictionaryKeys.jetpackModuleRequired)
-                    pub.jetpackSupport = dict.number(forKey: ServiceDictionaryKeys.jetpackSupport).boolValue
-                    pub.label = dict.string(forKey: ServiceDictionaryKeys.label)
-                    pub.multipleExternalUserIDSupport = dict.number(forKey: ServiceDictionaryKeys.multipleExternalUserIDSupport).boolValue
-                    pub.type = dict.string(forKey: ServiceDictionaryKeys.type)
+                    pub.connectURL = dict.string(forKey: ServiceDictionaryKeys.connectURL) ?? ""
+                    pub.detail = dict.string(forKey: ServiceDictionaryKeys.description) ?? ""
+                    pub.externalUsersOnly = dict.number(forKey: ServiceDictionaryKeys.externalUsersOnly)?.boolValue ?? false
+                    pub.icon = dict.string(forKey: ServiceDictionaryKeys.icon) ?? ""
+                    pub.serviceID = dict.string(forKey: ServiceDictionaryKeys.ID) ?? ""
+                    pub.jetpackModuleRequired = dict.string(forKey: ServiceDictionaryKeys.jetpackModuleRequired) ?? ""
+                    pub.jetpackSupport = dict.number(forKey: ServiceDictionaryKeys.jetpackSupport)?.boolValue ?? false
+                    pub.label = dict.string(forKey: ServiceDictionaryKeys.label) ?? ""
+                    pub.multipleExternalUserIDSupport = dict.number(forKey: ServiceDictionaryKeys.multipleExternalUserIDSupport)?.boolValue ?? false
+                    pub.type = dict.string(forKey: ServiceDictionaryKeys.type) ?? ""
```

Comment:
The default values used above are the same as [RemotePublicizeService's](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/146821b4364723b00392cbdcdb452f2036f158f6/WordPressKit/RemotePublicizeService.swift) initial value.

Reply:
Wow... Such Objective-C "don't worry if the value's not there, it'll be fine" way of working 😄 

Adding more strictness to the library will be a whole new can of worms... if/when we'll get to it.